task_geo/data_sources/hdx_oxfd/datapackage.json

```diff
@@ -0,0 +1,169 @@
+                        "format": "default"
+                    },
+                    {
+                        "name": "s6_restrictions on internal movement",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "s6_isgeneral",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "s7_international travel controls",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "s8_fiscal measures",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "s9_monetary measures",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "s10_emergency investment in health care",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "s11_investment in vaccines",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "s12_testing framework",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "s13_contact tracing",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "confirmedcases",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "confirmeddeaths",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "stringencyindex",
+                        "type": "number",
+                        "format": "default"
+                    },
+                    {
+                        "name": "stringencyindexfordisplay",
+                        "type": "number",
+                        "format": "default"
+                    }
+                ],
+                "missingValues": [
+                    ""
+                ]
+            }
+        }
+    ],
+    "id": "45331aaa-e651-4a16-a8a0-92a51b3734c0",
+    "size": 265019,
+    "revision_last_updated": "2020-04-14T10:03:25.204286",
+    "last_modified": "2020-04-08T13:36:10.652755",
+    "created": "2020-04-03T08:47:53.859260",
```

Comment:
Please leave the id blank as it will be added during the datapackage creation process.

The revision, last modified and creaded, please append `data_source` to them